functions/src/Services/Event.ts

```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
+/**
+ * This function is used to easily handle retrieving the event collection with the converter
+ */
+function getEventCollection() {
+	return firestore().collection('events').withConverter(eventConverter);
+}
+
 export const createEvent = (event: Event) => {
-    const eventCollection = firestore().collection('events').withConverter(eventConverter);
+	const eventCollection = getEventCollection();
 	eventCollection.add(event);
 };
 
+/**
+ * This function currently does not work with the eventConverter, hence the need for the spread operator ({ ...event })
```

Comment:
You can be more specific
```suggestion
 * The update method does not work with the event converter, so `event` is unpacked and passed in as a separate object.
```

Reply:
It should instead say that the event converter only works with set and get, so event is unpacked and passed in as a separate object.

Reply:
Sounds good boss